tensorflow/core/kernels/sparse_xent_op.cc

```diff
@@ -65,7 +65,7 @@ class SparseSoftmaxXentWithLogitsOp : public OpKernel {
     OP_REQUIRES(context, logits.dim_size(0) == labels.dim_size(0),
                 errors::InvalidArgument(
                     "logits and labels must have the same first dimension, "
-                    "got logits shape ",
+                    "got logits shape " ,
```

Comment:
Nit: Please remove the extra space before comma.

Reply:
@penpornk Please ignore this PR, there have issues with my git config, thats why some changes, do not show up with correct id, I am creating new PR, then I will close this one.

Reply:
Got it. Please tag me in the new PR when it's ready. Thank you very much!